kickstarter/android-oss
# MBL-795: RewardsFragment Screen

# 📲 What

- Migration RXJava2 + ViewModel, take a look at the [engineering plan](https://kickstarter.atlassian.net/wiki/spaces/NT/pages/2350317579/ViewModel+Tech+Debt+Epic) if any doubt 

# 👀 See

https://github.com/kickstarter/android-oss/assets/4083656/101e41c6-7267-4087-923d-8f56a63581a8



| --- | --- |
|  |  |

# 📋 QA
- Browse several projects open the rewards carousel for addons, rewards, as backer, ... etc.
- Browse several finalized projects open the rewards carousel if no backer.
- Browse several finalized projects as backer, the manage pledge screen should appear.

# Story 📖

[MBL-795](https://kickstarter.atlassian.net/browse/MBL-795)


[MBL-795]: https://kickstarter.atlassian.net/browse/MBL-795?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
